### PR TITLE
feat(nlp): update PyTorch alt NLP to FastText

### PR DESCRIPTION
**Pull request by athewsey**
*Issue #, if available:* #1

*Description of changes:*

- Update PyTorch alternative for NLP exercise to align to the FastText word embeddings now used in the TF.Keras version.
- Tweak eval batch size for faster compute
- Adjust loss metrics for better train/val consistency ('sum over batch' was actually taking mean)
- Align default instance type to g4dn.xlarge
- Tweak TF.Keras example to use same `tokenize_*and*_pad_docs` function name and use same numpy one-hot implementation as PyTorch - rather than relying on TF dependency.
- Remove unnecessary imports

<br/>

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yudho** [comment]
Tested successfully on these scenarios:
Tensorflow Local Studio CPU kernel OK
Tensorflow Sagemaker Studio CPU kernel OK
PyTorch  Local Studio CPU kernel OK
PyTorch  SageMaker Studio CPU kernel OK
Tensorflow  Local Notebook CPU instance OK
Tensorflow  Sagemaker Notebook CPU instance OK
PyTorch  Local Notebook CPU instance OK
PyTorch  SageMaker Notebook CPU instance OK

The prediction results are also reasonable. 

The test does not include running it in local GPU environment for neither Studio nor Notebook.

The changes look good.
